suite-native/atoms/src/Card/Card.tsx

```diff
@@ -10,16 +10,17 @@ import { AlertBox, AlertBoxProps } from '../AlertBox';
 export type CardProps = {
     children: ReactNode;
     style?: NativeStyleObject;
+    noPadding?: boolean;
```

Comment:
what about
```suggestion
    isZeroPadding?: boolean;
```

Reply:
It's feels strange even if it's correct according to our code style guide. IDK